benchmarks/src/main/java/org/elasticsearch/benchmark/tdigest/SortBench.java

```diff
@@ -58,22 +60,22 @@ public class SortBench {
     public void setup() {
         Random prng = new Random(999983);
         for (int i = 0; i < size; i++) {
-            values[i] = prng.nextDouble();
+            values.set(i, prng.nextDouble());
         }
         if (sortDirection > 0) {
-            Arrays.sort(values);
+            values.sort();
         } else if (sortDirection < 0) {
-            Arrays.sort(values);
-            Sort.reverse(values, 0, values.length);
+            values.sort();
+            Sort.reverse(values, 0, values.size());
         }
     }
 
     @Benchmark
-    public void quicksort() {
-        int[] order = new int[size];
+    public void stableSort() {
+        TDigestIntArray order = WrapperTDigestArrays.INSTANCE.newIntArray(size);
         for (int i = 0; i < size; i++) {
-            order[i] = i;
+            order.set(i, i);
         }
-        Sort.sort(order, values, null, values.length);
+        Sort.stableSort(order, values, values.size());
```

Comment:
Did this spit out different performance numbers before and after?

Reply:
If you run the thing before and after you can make a little chart like I did in https://github.com/elastic/elasticsearch/pull/112717. It feels nice to have numbers to look at stored in github.

Reply:
Nice, gonna do that. I guess you did the "->" manually right? In case there's a simpler way to do that

Reply:
Added the 2 relevant benchmarks (TDigest and Sort) to the description

Reply:
> Nice, gonna do that. I guess you did the "->" manually right? In case there's a simpler way to do that

Yes. I stuff both examples into vscode and grab the output columns from one and stick them into another. I tend to trim a few other bits of the output while I'm there.